ApplicationDevelopmentLifecycle.md

```diff
@@ -8,4 +8,7 @@
 
 **TODO**
 
-- [ ] Add overview
\ No newline at end of file
+- [ ] Add overview
+- [ ] Flow should be clockwise
```

Comment:
Looks like there is a bug in the tex-package I used, as I configured it to be clockwise from the start:

https://github.com/blochberger/developer-guidelines/blob/a1d0ad659792dffbaa26e58f943f8b0cdd8179d5/figures/applifecycle.tex#L23

So the whole graphic needs to be redone.

Reply:
Fixed in #16.